ext_mongodb.php

```diff
@@ -878,6 +878,22 @@ public function __construct(int $readPreference, array $tagSets = null, array $o
 
 				break;
 
+			case "primary":
```

Comment:
Per https://github.com/mongodb/mongo-php-driver/pull/570/files#r109191024, shall we make this case-insensitive? Likewise a question for majority write concern.